### PR TITLE
Clarify wording on installation so issues are triaged more effectively

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

**Comment by sdake** [comment]
@sebastienvas this repo seems to be missing the bot.  Not sure if we care or not as there are not alot of files in it.

Cheers
-steve

**Review by ldemailly — COMMENTED** [review]
lol
no I don’t think mature/immature is appropriate

**Comment by ldemailly** [comment]
no bots are needed here, only 1 file

File: issue_template.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ kubectl version
 ```
 
 **Is Istio Auth enabled or not ?**
-Did you install the very mature istio.yaml, istio-auth.yaml....  or the immature Helm chart directly?
+Did you install the stable istio.yaml, istio-auth.yaml....  or the Helm chart?
```
Comment (ldemailly) [inline issue_template.md]: maybe ask their helm version for helm?

**Review by ldemailly — APPROVED** [review]

File: issue_template.md (hunk 1 above)
Comment (sdake) [reply issue_template.md]: Our docs specify a hard minimum on Helm and Helm is generally really stable as a tooling system at this point.  I think what would be more interesting data is if they used specific --set flags, but that opens pandora's box, especially in the official triage page.  Easier to ask that as a followup once triaged I think.

Cheers
-steve

Comment (ldemailly) [reply issue_template.md]: maybe ask for helm their full command line and output ? (if it's not too verbose)

anyway you can merge as is too, it's a good incremental improvment

**Comment by sdake** [comment]
command line input is good idea.  output would be unhelpful most of the time unless `-d` is used, and in that case, it would be better suited to a gist or paste.

**Comment by ldemailly** [comment]
btw I fixed up the settings on this repo, next PR should be a squash and merge like elsewhere
and I protected master too while at it

**Pull request merged**